src/infer_request.cc

```diff
@@ -682,6 +714,72 @@ InferenceRequest::Normalize()
 {
   const inference::ModelConfig& model_config = model_raw_->Config();
 
+  // Fill metadata for raw input
+  if (!raw_input_name_.empty()) {
+    if ((original_inputs_.size() != 1) || (model_config.input_size() != 1)) {
```

Comment:
I would suggest creating a local boolean before this if-statement. Something like,
```
...
if (!raw_input_name_.empty()) {
    bool input_size_not_one = (original_inputs_.size() != 1) || (model_config.input_size() != 1);
    if (input_size_not_one) {
...
```
Feel free to reject my naming convention in favor of your own. 

The reason for this is that when glancing over the code in the future, you don't want to have the overhead of having to reason out an if-statement before moving on in your train of thought. During debugging, this makes inspection of the logic easier as you can put a breakpoint directly on the temporary var. During a release build this temp var _should_ get optimized out by the compiler so there is no performance overhead either.